lib/engines/content_object_store/app/components/content_object_store/content_block_edition/index/summary_card_component.rb

```diff
@@ -0,0 +1,25 @@
+class ContentObjectStore::ContentBlockEdition::Index::SummaryCardComponent < ContentObjectStore::ContentBlockEdition::Show::SummaryListComponent
+private
+
+  def rows
+    items.map do |item|
+      {
+        key: item[:field],
+        value: item[:value],
+      }
+    end
+  end
+
+  def title
+    content_block_edition.document.title
+  end
+
+  def summary_card_actions
+    [
+      {
+        label: "View/edit",
```

Comment:
I haven't seen the prototype, this feels unusual to say just "View" so I wanted to checking it's deliberate.

Reply:
Yeah, this roughly chimes with what's in the prototype. We can always tweak later anyhoo